docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

```diff
@@ -1,5 +1,24 @@
 # This document lists known breaking changes in Roslyn after .NET 8 all the way to .NET 9.
 
+
+## InlineArray attribute on a record struct type is no longer allowed.
+
+***Introduced in Visual Studio 2022 version 17.12***
```

Comment:
I think if you merge now it will go into VS 17.11, but that's probably not very important.